test-data/unit/check-kwargs.test

```diff
@@ -115,13 +115,20 @@ main:4: error: Unexpected keyword argument "b"
 [case testKeywordOnlyArguments]
```

Comment:
Add a copy of this test case that uses the fast parser (`# flags: --fast-parser` in the test case).